code/utility/src/palindrome/palindrome_check/Palindrome.py

```diff
@@ -0,0 +1,39 @@
+
+def main():
+    """
+    Determine whether or not a string is a palindrome.
+    :return: 'That's a palindrome.' if the string is a palindrome,
```

Comment:
I think this shouldn't be written with `:return`, because it is just print.

Reply:
done